oarepo_requests/ui/theme/assets/semantic-ui/js/oarepo_requests_ui/common/utils/hooks.js

```diff
@@ -16,7 +21,45 @@ export const useAction = ({
   modalControl,
   requestActionName,
 } = {}) => {
-  const { onBeforeAction, onAfterAction, onActionError } = useCallbackContext();
+  const {
+    onBeforeAction,
+    onAfterAction,
+    onErrorPlugins = [],
+    actionExtraContext,
+  } = useCallbackContext();
+  const handleActionError = (e, variables) => {
+    const context = {
+      e,
+      variables,
+      formik,
+      modalControl,
+      requestOrRequestType,
+      requestActionName,
+      actionExtraContext,
+    };
+
+    for (const plugin of onErrorPlugins) {
+      const handled = plugin(e, context);
+      if (handled) {
+        return;
+      }
+    }
+
+    const defaultPlugins = [
```

Comment:
merge default plugins to end of `onErrorPlugins`, remove following code, to keep function short